internal/pkg/config/config.go

```diff
@@ -39,14 +51,20 @@ var DefaultOptions = []interface{}{
 	ucfg.VarExp,
 	VarSkipKeys("inputs"),
 	ucfg.IgnoreCommas,
+	OTelKeys("receivers", "processors", "exporters", "extensions", "service"),
```

Comment:
missing `connectors` 
refer to unmarshaller https://github.com/open-telemetry/opentelemetry-collector/blob/8e522ad950de6326a0841d7e1bef808bbc0d3537/otelcol/unmarshaler.go#L18